rotkehlchen/data/importer.py

```diff
@@ -792,10 +792,13 @@ def _consume_nexo(self, csv_row: Dict[str, Any]) -> None:
         - UnknownAsset
         - DeserializationError
         """
-        if 'rejected' not in csv_row['Description']:
+
+        ignored_entries = ('ExchangeToWithdraw', 'DepositToExchange')
+
```

Comment:
unnecessary whitespace

```suggestion
        ignored_entries = ('ExchangeToWithdraw', 'DepositToExchange')
```